docs/contributing/developing.rst

```diff
@@ -116,6 +116,35 @@ If your Elasticsearch instance is located somewhere else, you can set the
 
     $ ELASTICSEARCH_URL=http://my-elasticsearch-instance:9200 python runtests.py --elasticsearch
 
+**Browser and device support**
+
+Wagtail is meant to be used on a wide variety of devices and browsers. Supported browser / device versions include:
+
+=============  =============  =============
+Browser        Device/OS      Version(s)
+=============  =============  =============
+Mobile Safari  iOS Phone      Latest
+Mobile Safari  iOS Tablet     Latest
+Chrome         Android        Latest
```

Comment:
I agree on principle, but realistically I personally only have access to one test iPhone and one test iPad. I think that's something to be discussed further – whether it's ok to have a difference between support targets and what you actually test things on.